webrtc.html

```diff
@@ -5121,8 +5121,7 @@ <h2>RTP Media API</h2>
     <code><a data-link-for="RTCRtpSender">replaceTrack</a></code> changes the
     track sent by an <code><a>RTCRtpSender</a></code> without creating a new
     track on the receiver side; the corresponding
-    <code><a>RTCRtpReceiver</a></code> will only have a single track,
-    potentially representing multiple sources of media stitched together. Both
+    <code><a>RTCRtpReceiver</a></code> will only have a single track. Both
```

Comment:
I think we should explain what happens. To me, this is a high-level note explaining the model, not instructions to an implementer. I was never confused by it, but if "stitched together" is confusing people, we should use other words to describe the net result.

How about referring to the observable settings just being added in https://github.com/w3c/webrtc-pc/pull/2188?

*"will only have a single track, composed of the RTP data received. This means that properties like* `width`, `height` *and* `frameRate` *may change as a result."*?

Reply:
"Properties like width, height and frameRate may change over the lifetime of the track, based on the RTP data received."

Reply:
@jan-ivar "stitched together" is not _confusing_. You are correct in your analysis that the issue filed https://github.com/w3c/webrtc-pc/issues/2171 is asking for how the procedure happens. That is, how to reproduce the effect of `replaceTrack` without necessarily using `RTCRtpSender.replaceTrack`. Ultimately, the requirement for own code is to not use `RTCPeerConnection` at all while still having the ability to replace a `MediaStreamTrack` that is defined within a `MediaStream` passed to `MediaRecorder` while the `MediaRecorder` instance is active - at both Firefox and Chromium. Browsed the source code at Firefox and Chromium though not sure how to replicate the functionality of `replaceTrack`; perhaps create a `Proxy` object for each `MediaStreamTrack` which returns the same `id` for each respective track that is expected to be "stitched together"?

Reply:
@jan-ivar E.g., what would be necessary for `replaceTrack` to be implemented as a global method that can be used for any `MediaStream` or `MediaRecorder` instance to replace a given `MediaStreamTrack` - without having to use `RTCPeerConnection` code simply for the purpose of using `RTCRtpSender.replaceTrack()`?

Reply:
@aboba Should make it clear here and now that the language "multiple sources of media stitched together" and "have the sender switch seamlessly to transmitting withTrack instead of the sender's existing track" is not confusing - the issue did not state that the language should be removed. That language is what lead this user to experiment with `RTCPeerConnection` within the scope of attempting to record multiple media fragments to a single media file using `MediaRecorder` (specifically, not using any other third-party API than what is shipped with the ostensibly FOSS browsers Chromium and Firefox). The issue was filed to ask for the complete procedure used for `replaceTrack`, for the purpose of "reverse engineering" the procedure, and/or for the ability to file a PR at media-capture-record (or other appropriate repository) which uses the outline of how `replaceTrack` works to have no confusion as to what am requesting at media-capture-record, to address the long-standing feature requests of being able to record multiple media fragments or complete tracks (specifically, video tracks) using `MediaRecorder` (https://github.com/w3c/mediacapture-record/issues/4; https://github.com/w3c/mediacapture-record/issues/147; https://github.com/w3c/mediacapture-record/issues/166; https://bugs.chromium.org/p/chromium/issues/detail?id=528523; https://bugzilla.mozilla.org/show_bug.cgi?id=1276928).